triton-server/src/notification.ts

```diff
@@ -146,4 +210,110 @@ const mockDataset: Dataset = {
     released_status_count: 99,
     blocked_status_count: 64,
     latest_release_update: new Date().toISOString(),
+    archived_comments: [],
+    validation_status: 0,
+    validated_by: 64,
+    latest_validation_update: new Date().toISOString(),
+}
+
+const getValidationFlagLabel = (status: number) => {
+    switch (status) {
+        case 0:
+            return "Available"
+        case 1:
+            return "Passed"
+        case 2:
+            return "Failed"
+        default:
+            break
+    }
+}
+
+// this should also do the api call to get the basic info from the user
+const extractValidatedDatasetsInfo = async (validatedDataset: Dataset[]) => {
+    let extractedData: ExtractedValidatedNotificationData[] = []
+    validatedDataset.map((item: Dataset) => {
+        let runsInfo: ProjectAndRunInfo
+        let userCommentInfo: BasicCommentUserInfo | undefined = undefined
+        runsInfo = {
+            run_name: item.run_name,
+            project_name: item.project_name,
+            project_id: item.external_project_id,
+            lane_number: item.lane,
+            validation_status: item.validation_status,
+            validated_by: item.validated_by ?? "",
+            latest_validation_update: item.latest_validation_update,
+        }
+        if (item.archived_comments.length > 0) {
+            userCommentInfo = {
+                comment: item.archived_comments[0].comment,
+                created_at: item.archived_comments[0].created_at,
+                user_id: item.archived_comments[0].created_by,
+            }
+        }
+        extractedData.push({
+            basicCommentUserInfo: userCommentInfo,
+            projectAndRunInfo: runsInfo,
+        })
+    })
+
+    return extractedData
+}
+
+interface BasicCommentUserInfo {
+    user_id: number
+    created_at: string
+    comment: string
+    name?: string
+}
+
+interface ProjectAndRunInfo {
+    run_name: string
+    lane_number: number
+    validated_by?: string | number
+    name?: string
+    project_name: string
+    validation_status: ValidationFlag
+    latest_validation_update: string
+    project_id?: string
+}
+
+interface ExtractedValidatedNotificationData {
+    projectAndRunInfo: ProjectAndRunInfo
+    basicCommentUserInfo?: BasicCommentUserInfo
+}
+
+export const sendValidationEmail = async (
+    validatedDatasets: ExtractedValidatedNotificationData[],
+    body: string,
+    db: any,
+) => {
+    if (validatedDatasets.length > 0) {
+        let lastDate: string | undefined = undefined
+        for (const dataset of validatedDatasets) {
+            if (dataset.projectAndRunInfo.validation_status > 0) {
+                const subject = `A Run has been validated.`
+                await sendEmail(
+                    "",
+                    "sequencing-runs@computationalgenomics.ca",
```

Comment:
I believe we should use environment variable.

Reply:
maybe  let's call it VALIDATION_NOTIFICATION_EMAIL?

Reply:
as to leave a trace of our convo on slack, i'll create another pr to include the changes for the env variables